### ISSUE TITLE
Don't lock wasm-pack packages to nightly

### ISSUE THREAD

**Issue by clanehin** [rustwasm/wasm-pack#217]
Currently, wasm-pack seems to call cargo build with +nightly. This may not be the user's intention, especially in "no-install" mode.

If I'm not mistaken we can set the toolchain using `rustup override set nightly` immediately after installing it. Then if we installed the toolchain we'll automatically use it in every subsequent command, and if we didn't then whatever goes wrong will be the user's responsibility. :)

If no one objects or gets to it before me I'll submit a pull request.

**Comment by ashleygwilliams** [comment]
hey @clanehin ! at the moment, wasm-bindgen only works on nightly, so in order to build we'd need nightly still. once the 2018 edition lands, we should be able to move wasm-bindgen to stable, and then i would absolutely be interested in this :)

**Comment by clanehin** [comment]
> wasm-bindgen only works on nightly

Yep, but it doesn't have to be +nightly specifically. It will work fine with reasonably recent nightly releases like "nightly-2018-06-24".

**Comment by ashleygwilliams** [comment]
we;re going to address this #289, closing this in favor of the most recent one
